Goose.Tests/Application.IntegrationTests/issues/IssueConversationTests.cs

```diff
@@ -1,265 +1,272 @@
-﻿using Goose.API;
-using Goose.API.Repositories;
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Goose.API;
 using Goose.API.Services.Issues;
 using Goose.Domain.DTOs;
 using Goose.Domain.DTOs.Issues;
-using Goose.Domain.Models.Auth;
-using Goose.Domain.Models.Identity;
 using Goose.Domain.Models.Issues;
 using Goose.Domain.Models.Projects;
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.Extensions.DependencyInjection;
 using NUnit.Framework;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using System.Net.Http.Headers;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Goose.Tests.Application.IntegrationTests.issues
 {
     [TestFixture]
-    [SingleThreaded]
+    [Parallelizable(ParallelScope.All)]
     class IssueConversationTests
     {
-        private HttpClient _client;
-        private IIssueRequirementService _issueRequirementService;
-
-        [OneTimeSetUp]
-        public void OneTimeSetUp()
+        private sealed class TestScope : IDisposable
```

Comment:
Wieso wird TestScope in jeder Testdatei neu definiert? Kann man das nicht an einer Stelle zentral machen?

Reply:
Das ist Willy auch schon aufgefallen und arbeitet gerade daran.